noise_raster/constants.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+"""
+/***************************************************************************
+ NoiseRaster constants
+ ***************************************************************************/
+"""
+import tempfile
+import os
+
+REPROJECTED_TIF3035 = "_3035.tif"
+
+REPROJECTED_TIF25832 = "_25832.tif"
+
+REPROJECTED_SHP3035 = "_3035.shp"
+
+REPROJECTED_SHP25832 = "_25832.shp"
+
+logger = None
+
+root_temp_dir = tempfile.gettempdir().replace(os.sep, '/')
```

Comment:
`root_temp_dir` should be capitalized as it's used as a constant.

Reply:
I can capitilize it.